tests/spec/auth/redirect/api.js

```diff
@@ -140,6 +140,36 @@ function (bdd, assert, RedirectBroker, WindowMock, sinon, p) {
         });
       });
     });
+
+    bdd.describe('bestChoice', function () {
+      bdd.it('should reject if `scope` is not specified', function () {
```

Comment:
Not for this PR, but since testing for required parameters is done in all of the tests, and the code is largely boilerplate, maybe we should just create one test helper that can be included that loops over the list of required options and tests each one.